config/metadata/rulesMeta.yaml

```diff
@@ -402,6 +402,111 @@ groups:
           is a useful indicator of traces you'd like to see in Honeycomb, set
           this to true.
 
+  - name: WindowedThroughputSampler
```

Comment:
Do we even need the ThroughputSampler anymore?

Reply:
Yes, mainly for compatibility. I do hope to deprecate it but don't think we need to do that for 2.0.